installers/rpm-code/codes/elegant.sh

```diff
@@ -157,7 +161,7 @@ elegant_main() {
         tcsh
     codes_dependencies common
     elegant_download
-    elegant_build
+    elegant_build ${1:-}
```

Comment:
This argument should be called out with a name: gpu_only.

If the `return` above works, then you would want to pass `elegant_install_bin "$gpu_only"` (which, since internal, can just be a 1 or empty). Then inside elegant_install_bin, you would only install gpu-elegant.

Reply:
f4c99d0 is this right? It isn't what you described. I just pass in the whole arg array. But seems simpler to me than dealing with return from bash functions to create the $gpu_only only to pass it in and then have to test again.